Congo_DR/COD_transform_load_dlt.py

```diff
@@ -159,8 +159,7 @@ def boost_silver():
             .when(col('geo1').isin('Katanga', 'Tanganyika', 'Haut Lomami', 'Lualaba', 'Haut Katanga'), 'Katanga')
             .when(col('geo1').isin('Kinshasa', 'Ville Province De Kinshasa'), 'Kinshasa')
             .when(col('geo1').isin('Nord Kivu'), 'Nord-Kivu')
-            .when(col('geo1').isin('Ituri', 'Haut Uele', 'Orientale', 'Bas Uele', 'Tshopo'), 'Orientale')
-            .otherwise(None)
+            .when(col('geo1').isin('Ituri', 'Haut Uele', 'Orientale', 'Bas Uele', 'Tshopo', 'Maniema'), 'Orientale')
```

Comment:
Maniema should be its own region according to wikipedia: https://en.wikipedia.org/wiki/Democratic_Republic_of_the_Congo#Administrative_divisions. Can you double check and revise?

Reply:
Here is the updated code, with the mapping to the 11 provinces prior to change:

.withColumn('geo_0.5', 
    when(col('geo1').isin('Kwilu', 'Mai Ndombe', 'Kwango'), 'Bandundu')
    .when(col('geo1').isin('Kongo Central'), 'Bas Congo')
    .when(col('geo1').isin('Sud Ubangi', 'Nord Ubangi', 'Equateur', 'Tshuapa', 'Mongala'), 'Equateur')
    .when(col('geo1').isin('Kasai Central', 'Kasai'), 'Kasai Occidental')
    .when(col('geo1').isin('Lomami', 'Sankuru', 'Kasai Oriental'), 'Kasai Oriental')
    .when(col('geo1').isin('Tanganyika', 'Haut Lomami', 'Lualaba', 'Haut Katanga'), 'Katanga')
    .when(col('geo1').isin('Ville Province De Kinshasa'), 'Kinshasa')
    .when(col('geo1').isin('Ituri', 'Haut Uele', 'Bas Uele', 'Tshopo'), 'Orientale')
    .when(col('geo1').isin('Nord Kivu'), 'Nord Kivu')
    .when(col('geo1').isin('Maniema'), 'Maniema')
    .when(col('geo1').isin('Sud Kivu'), 'Sud Kivu')